docs/guides/set_up.md

```diff
@@ -10,29 +10,46 @@ Ensure that your machine satisfies the following requirements:
 - [Docker Engine](https://docs.docker.com/engine/install/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 
-
 ## Set up
 
 1. Create a workspace folder:
-```bash
-mkdir my_workspace
-cd my_workspace
-```
+
+    ```bash
+    mkdir my_workspace
+    cd my_workspace
+    ```
 
 2. Set up the environment. Remember to use the Python version you have installed. Here we are using 3.10 as reference:
-```bash
-touch Pipfile && pipenv --python 3.10 && pipenv shell
-```
+
+    ```bash
+    touch Pipfile && pipenv --python 3.10 && pipenv shell
+    ```
 
 3. Install the {{open_autonomy}} framework:
-```bash
-pip install open-autonomy[all]
-```
+
+    ```bash
+    pip install open-autonomy[all]
+    ```
```

Comment:
Should be installed with the `pipenv install` since we're working with a virtual env

Reply:
This is fine, it's already from within the shell